src/template.html

```diff
@@ -0,0 +1,725 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Repair design project</title>
+</head>
+
+<body>
+  <header class="home-header">
+    <nav class="nav">
+      <a class="logo nav__logo" href="#">
+        <h1>IC "RERAIR DESIGN PROJECT"</h1>
```

Comment:
Looks like typo. As i suggested earlier using some VSCode extensions can help to avoid such mistakes.